### PR TITLE
Add bind command using SBO SDK

### PR DESCRIPTION
**Pull request by wtrocki**
Add new command that uses SBO SDK to perform binding.

### How this works

Comands list your apps (or let you provide it's name) and then binds one of your current or seletect managed kafkas.

### Limitations

- Command will be in preview mode
- Can be mentioned in quickstarts but will not be used during summit
- Needs to have preview label
- We cannot touch/refactor the existing `connect` command for stability etc.

## Verification

Happy day scenario without arguments
```
[wtrocki@graphapi app-services-cli (service-binding-sdk ✗)]$ rhoas cluster bind                                                                                   
Namespace not provided. Using  wtrocki-quarkus-env
? Please select application you want to connect with rhoas-kafka-quickstart-example
Binding 'youcan-use-your-cluster' with 'rhoas-kafka-quickstart-example' app 
? Do you want to continue? Yes
Binding youcan-use-your-cluster with rhoas-kafka-quickstart-example app succeeded
```

Happy day scenario with explicit values
```
[wtrocki@graphapi app-services-cli (service-binding-sdk ✗)]$  rhoas cluster bind --namespace wtrocki-quarkus-env --appName=rhoas-kafka-quickstart-example 
Binding 'youcan-use-your-cluster' with 'rhoas-kafka-quickstart-example' app 
? Do you want to continue? Yes
Binding youcan-use-your-cluster with rhoas-kafka-quickstart-example app succeeded
```

Unauthorized in Kubernetes
```
[wtrocki@graphapi app-services-cli (service-binding-sdk ✗)]$ rhoas cluster bind                 
Namespace not provided. Using  wtrocki-quarkus-env
Error: Unauthorized
```

Selecting Kafka (that might execute auto-connect in future (but will require changes in connect to do so)
```
[wtrocki@graphapi app-services-cli (service-binding-sdk ✗)]$ rhoas cluster bind --ignore-context 
? Select Kafka instance to connect: sidhax-offsec
Namespace not provided. Using  wtrocki-quarkus-env
? Please select application you want to connect with example
Binding 'sidhax-offsec' with 'example' app 
? Do you want to continue? Yes
Error: Selected Kafka doesn't exist on the server. Please execute rhoas cluster connect first 
```

### CODE REVIEW

**Comment by wtrocki** [comment]
Still WIP. I will hold with i18n refactoring until review is made - that will make things easier.

File: pkg/cmd/cluster/bind/bind.go [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+					TemplateData: map[string]interface{}{
+						"Flag": "appName",
+					},
+				}))
+			}
+			return runBind(opts)
+		},
+	}
+
+	cmd.Flags().StringVarP(&opts.kubeconfigLocation, "kubeconfig", "", "", localizer.MustLocalizeFromID("cluster.common.flag.kubeconfig.description"))
```
Comment (craicoverflow) [inline pkg/cmd/cluster/bind/bind.go:66]: There should also be a `-y` flag, to enable this to be run in a CI environment.

**Review by craicoverflow — APPROVED** [review]
Let some comments, but not crucial for now since this is a preview.

File: pkg/cluster/serviceBinding.go [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+		return fmt.Errorf("Selected Kafka doesn't exist on the server. Please execute rhoas cluster connect first ")
+	}
+
+	// Execute binding
+	err = performBinding(serviceName, appName, ns, clients)
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("Binding %v with %v app succeeded\n", serviceName, appName)
```
[the remark this answers is not in the document]
Comment (wtrocki) [reply pkg/cluster/serviceBinding.go:90]: Yes. I left all of that for i18n. Doing this now

File: pkg/cmd/cluster/bind/bind.go [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+		Config:     f.Config,
+		Connection: f.Connection,
+		Logger:     f.Logger,
+		IO:         f.IOStreams,
+	}
+
+	cmd := &cobra.Command{
+		Use:     "bind",
+		Short:   "Connect your service with your application",
+		Long:    `[Beta] Command allows you to connect services created by connect command to your application`,
```
[the remark this answers is not in the document]
Comment (wtrocki) [reply pkg/cmd/cluster/bind/bind.go:43]: Actually we do not need include this. 
The only side effect of this command is that will not work with DeploymentConfigs. Created follow up issue for this

**Comment by wtrocki** [comment]
Added language files. Verified again with all cases as above

File: pkg/cluster/serviceBinding.go [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+		}
+	} else {
+		_, err = clients.dynamicClient.Resource(deploymentResource).Namespace(ns).Get(context.TODO(), appName, metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+	}
+
+	// Status and confirmation
+	fmt.Printf("Binding '%v' with '%v' app \n", serviceName, appName)
```
[the remark this answers is not in the document]
Comment (wtrocki) [reply pkg/cluster/serviceBinding.go:69]: Missed that one :D

**Review by craicoverflow — APPROVED** [review]

**Pull request merged**